pkg/converter/converter_test.go

```diff
@@ -581,7 +581,7 @@ func TestConvertConfigs(t *testing.T) {
 
 	properties := map[string]map[string]string{
 		configId: {
-			"name":                 configName,
+			"name":                 fmt.Sprintf("%s {{ .Env.%s }}", configName, envVariableName),
```

Comment:
nit: do we really want to keep stuffing additional cases into the same test? Might be nicer to add a dedicated test for this at least this very specific case.